sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,8 @@
+-- Proc Chaos Strike only by its two damage spells and only if they are critical hits
+DELETE FROM `spell_proc_event` WHERE `entry`= 197125;
+INSERT INTO `spell_proc_event` (`entry`, `SchoolMask`, `SpellFamilyName`, `SpellFamilyMask0`, `SpellFamilyMask1`, `SpellFamilyMask2`, `spellFamilyMask3`, `procFlags`, `procEx`, `ppmRate`, `CustomChance`, `Cooldown`) VALUES
```

Comment:
The table `spell_proc_event` does not exist in `master` TDB. From travis-ci :
```
ERROR 1146 (42S02) at line 416593: Table 'world.spell_proc_event' doesn't exist
```
Maybe you meant `spell_proc` ?
```sql
(`SpellId`,`SchoolMask`,`SpellFamilyName`,`SpellFamilyMask0`,`SpellFamilyMask1`,`SpellFamilyMask2`,`SpellFamilyMask3`,`ProcFlags`,`SpellTypeMask`,`SpellPhaseMask`,`HitMask`,`AttributesMask`,`ProcsPerMinute`,`Chance`,`Cooldown`,`Charges`)
```

Reply:
check the date, when I commited this. It was back then, when the spell_proc table did not exist

Reply:
@Faq ^